aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/AWSPredictionsService.java

```diff
@@ -104,12 +107,19 @@ public void translate(
      * @param onError triggered upon encountering error
      */
     public void detectLabels(
-            @NonNull LabelType type,
+            @NonNull IdentifyAction type,
```

Comment:
Under what conditions is it valuable to provide a non-`LabelType` object, here? I see later you have some control flow around a `ClassCastException`.

Reply:
I figured it might be confusing to have people provide `IdentifyActionType` everywhere except when doing it for label detection and text detection, so I wanted `IdentifyActionType.DETECT_LABELS` to be a valid entry also. Besides, the configuration value will be always ignored if the user is forced to supply an explicit value here all the time, and I figured `IdentifyActionType.DETECT_LABELS` is a great choice to use as the default option.